src/pages/ForgotPassword/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Input from 'components/form/Input';
+import Button from 'components/common/Button';
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import useTranslation from 'hooks/useTranslation';
+
+import './style.css';
+import { useResetPasswordMutation } from 'services/profile/profile';
+
+const ForgotPassword = () => {
+  const t = useTranslation();
+  const schema = z.object({
+    email: z.string().email({ message: t('signup.errors.emailMsg') }),
+  });
+  const {
+    register: forgotPassowrd,
+    handleSubmit: handleOnSubmitForgotPassowrd,
+    formState: { errors: forgotPassowrdErrors },
+  } = useForm({ resolver: zodResolver(schema) });
+  const [isSend, setIsSend] = useState(false);
+  const [resetPassword] = useResetPasswordMutation();
```

Comment:
With redux toolkit, you could use isSuccess to validate when the mutation was successful.

https://redux-toolkit.js.org/rtk-query/usage/mutations#frequently-used-mutation-hook-return-values

```suggestion
  const [resetPassword, { isSuccess }] = useResetPasswordMutation();
```